grassroots-frontend/src/routeTree.gen.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable */
+
+// @ts-nocheck
+
+// noinspection JSUnusedGlobalSymbols
+
+// This file was automatically generated by TanStack Router.
+// You should NOT make any changes in this file as it will be overwritten.
+// Additionally, you should also exclude this file from your linter and/or formatter to prevent it from being checked or modified.
```

Comment:
If you add a `.gitattributes` file with the following, github will collapse this file by default.

```
**/*.gen.ts linguist-generated=true
```

Reply:
Done!